<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Ameet" failures="1" tests="11" name="testcases.RestAssured_TestCases" time="67.871" errors="0" timestamp="2025-07-11T11:59:23 IST" skipped="0">
  <testcase classname="testcases.RestAssured_TestCases" name="loginWithSeleniumAndGetCookie" time="34.257"/>
  <testcase classname="testcases.RestAssured_TestCases" name="getSubunitsTree" time="18.396"/>
  <testcase classname="testcases.RestAssured_TestCases" name="getAdminSkills" time="1.230"/>
  <testcase classname="testcases.RestAssured_TestCases" name="getAdminEdu" time="1.152"/>
  <testcase classname="testcases.RestAssured_TestCases" name="getAdminLicenses" time="1.027"/>
  <testcase classname="testcases.RestAssured_TestCases" name="getAdminLanguages" time="0.981"/>
  <testcase classname="testcases.RestAssured_TestCases" name="deleteValidUsers" time="2.572">
    <failure message="Expected status code 200 for success. expected [200] but found [422]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Expected status code 200 for success. expected [200] but found [422]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at testcases.RestAssured_TestCases.deleteValidUsers(RestAssured_TestCases.java:309)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- deleteValidUsers -->
  <testcase classname="testcases.RestAssured_TestCases" name="postNewJobTitles" time="4.522"/>
  <testcase classname="testcases.RestAssured_TestCases" name="putJobTitleByID" time="1.309"/>
  <testcase classname="testcases.RestAssured_TestCases" name="postUniquePaygrade" time="1.236"/>
  <testcase classname="testcases.RestAssured_TestCases" name="deletePaygradeById" time="1.189"/>
</testsuite> <!-- testcases.RestAssured_TestCases -->
